cmd/corectl/main.go

```diff
@@ -120,12 +133,15 @@ func configGenerator(db *sql.DB, args []string) {
 
 	conf := &config.Config{
 		IsGenerator: true,
-		IsSigner:    *isSigner,
 		Quorum:      quorum,
 		Signers:     signers,
 		MaxIssuanceWindow: chainjson.Duration{
 			Duration: *maxIssuanceWindow,
 		},
+		IsSigner:            *flagK != "",
```

Comment:
Just to double check, previously if `IsSigner` was passed true without `BlockPub`, then `config.Configure` would generate a block key. Are we fine losing that option?